extendr-api/src/error.rs

```diff
@@ -128,6 +130,10 @@ impl std::fmt::Display for Error {
             Error::OutOfRange(_robj) => write!(f, "Out of range."),
             Error::MustNotBeNA(_robj) => write!(f, "Must not be NA."),
             Error::ExpectedNonZeroLength(_robj) => write!(f, "Expected non zero length"),
+            Error::ExpectedIntegerish(robj) => {
+                write!(f, "Expected an integer-ish value, got {:?}", robj)
```

Comment:
Not sure if "integer-ish value" is understandable. Maybe "integer or a double representing a whole number"?

Reply:
Maybe write "float" or "numeric" instead of "double"?

Reply:
Thanks, sounds good. I picked the word "integer-ish" from [an rlang's function](https://rlang.r-lib.org/reference/is_integerish.html), but probably it's better to say "a whole number."